docs/content/en/docs/implementing/analysis.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Analyses
+description: Understand Analyses in Keptn and how to use them
+weight: 150
+---
+
+The Keptn Metrics Operator implements an SLO/SLI feature set inspired by Keptnv1 under the name of Analysis.
+With an Analysis Definition you can specify multiple Service Level Objectives that will be evaluated in your Analysis.  
+At the end of the Analysis the status will return whether your objective failed or passed.
+
+This data can be seen as prometheus metrics and can be displayed on Grafana.
```

Comment:
```suggestion
This data can be seen as Prometheus metrics and can be displayed on Grafana.
```

Reply:
If you'd like to include the name of the metrics here as well, they are:

- `keptn-analysis-result`: The overall score of each analysis, together with the time frame, name of the analysis definition and namespace

![Screenshot 2023-09-20 at 14 44 27](https://github.com/keptn/lifecycle-toolkit/assets/2143586/052dc5a4-d92e-4293-bd14-21d9a5632005)

- `keptn-objective-result`: the result of each objective (weight, keySLI, timefra,e, namespace, analysis instance name, value of the retrieved metric)
- 
![Screenshot 2023-09-21 at 09 29 10](https://github.com/keptn/lifecycle-toolkit/assets/2143586/c93fca50-8c70-4036-8797-7f7bbd8c95d1)